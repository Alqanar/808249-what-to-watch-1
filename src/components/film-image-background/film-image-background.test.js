import React from "react";
import renderer from "react-test-renderer";

import FilmImageBackground from "./film-image-background";


const filmMock = {
  name: `Avengers: Endgame`,
  coverLink: `https://st.kp.yandex.net/images/film_iphone/iphone360_843650.jpg`,
  posterLink: `https://st.kp.yandex.net/images/film_iphone/iphone360_843650.jpg`,
  genre: [`Sci-Fi`],
  released: `2019`
};

it(`Image of the featured film correctly renders`, () => {
  const tree = renderer
    .create(
      <FilmImageBackground
        film={filmMock}
      />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
