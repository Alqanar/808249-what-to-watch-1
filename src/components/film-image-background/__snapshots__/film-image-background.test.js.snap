// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image of the featured film correctly renders 1`] = `
<div
  className="movie-card__bg"
>
  <img
    alt="Avengers: Endgame"
    src="https://st.kp.yandex.net/images/film_iphone/iphone360_843650.jpg"
  />
</div>
`;
